ta/pkcs11/include/pkcs11_ta.h

```diff
@@ -249,6 +249,35 @@ enum pkcs11_ta_cmd {
 	 * This command relates to the PKCS#11 API function C_Logout().
 	 */
 	PKCS11_CMD_LOGOUT = 14,
+
+	/*
+	 * PKCS11_CMD_IMPORT_OBJECT - Import a raw client assmebled object in
```

Comment:
s/assmebled/assembled/

Better with a terminal dot at next line, I think.

Reply:
A terminal dot at the next line would be rather unique among the comments for this enum.
I'll fix the spell error.

Reply:
ok. I though that since this comment is a multi-line, a terminal dot makes it clear the end of the sentence is reached. That said, I'm fine both ways.